policy_set.go

```diff
@@ -122,7 +125,8 @@ const (
 	PolicySetCurrentVersion PolicySetIncludeOpt = "current_version"
 	PolicySetNewestVersion  PolicySetIncludeOpt = "newest_version"
 	// **Note: This field is still in BETA and subject to change.**
-	PolicySetProjects PolicySetIncludeOpt = "projects"
+	PolicySetWorkspaceExclusions PolicySetIncludeOpt = "workspace_exclusions"
```

Comment:
Can you add a test that reads a policy set and "Includes" workspace_exclusions? I want to make sure that the platform expects an underscore and not a dash.